### ISSUE TITLE
Request: Popdown

### ISSUE THREAD

**Issue by Spielmops** [TeamNewPipe/NewPipe#637]
I propose that the popup pops down at the end of the video.

Spielmops

**Comment by theScrabi** [comment]
Maybe we could add something for that in the settings.

**Comment by PacoBell** [comment]
Instead of placing an 'x' or something similar in the already crowded popover window, perhaps just have a floating 'x' target show up at the middle-bottom when the user moved the window an have it go away otherwise? I've seen such an interaction in other apps before and I personally find it quite pleasing.

_Sent from my Motorola XT1650 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by theScrabi** [comment]
Yea I know what you mean, sounds like an option. But was it worth the effort?
@mauriciocolli you Implemented that. What do you think?

**Comment by Spielmops** [comment]
Why not automatically at the end of the video?

**Comment by mauriciocolli** [comment]
@PacoBell seems like you're mixing issues here, please open another issue with your specific request.

> theScrabi > ...you Implemented that.

I don't think so.

--- 

But yes, I'd like to have this option.

One case come to mind, what if you had repeat enabled? (In my opinion, the popup wouldn't close then, otherwise it'd be a inconsistent behavior).

**Comment by Spielmops** [comment]
You think to far, let's keep this easy. Why should we enable repeat, if we can easily start again?

**Comment by mauriciocolli** [comment]
Well, it's already a option (in the notification), so I don't think we can just ignore it, do we?

**Comment by iiiGerardoiii** [comment]
>what if you had repeat enabled? (In my opinion, the popup wouldn't close then

Agree, just make it not pop down if repeat is enabled. Most video players don't close when they have repeat enabled so I think users will expect this behavior.

**Comment by Echolon** [comment]
I would really prefer a X in one conrner to close the PiP Window. often enough I dont watch the video till the end

**Comment by iiiGerardoiii** [comment]
@Echolon that's quite off-topic. You can do a swipe right gesture to close the video btw.

**Comment by SameenAhnaf** [comment]
How about closing the popup automatically (with a lag of 3 sec) if no action has taken place (just like YouTube auto-queue lag)?
![IMG_20210705_081349](https://user-images.githubusercontent.com/79245641/124408915-1b0be100-dd69-11eb-89e4-9038e86b75aa.jpg)
